share/module_check_a_mundo.F

```diff
@@ -591,6 +592,9 @@ END FUNCTION bep_bem_nbui_max
             count_fatal_error = count_fatal_error + 1
          END IF
       END IF
+#else
+      EXIT
```

Comment:
@smileMchen 
Ming,
I do not see why these two lines are required. It seems like they could be completely removed, since we only care about the status of the table if the source code is built for SBM.

Reply:
@davegill 
Dave,
We have a CPP flag that controls whether FSBM code will be compiled or not (BUILD_SBM_FAST). We only need to check the input tables when the FSBM is compiled ( BUILD_SBM_FAST==1). If BUILD_SBM_FAST == 0, one shouldn't turn on the FSBM scheme.

Reply:
Ming,
Thanks for the explanation. We already have that check in place (use mp_physics==30, but the code is NOT built) within check_a_mundo.

```
# if( BUILD_SBM_FAST != 1)
!-----------------------------------------------------------------------
!  If the FAST SBM scheme is requested and it is not compiled, let the
!  user know.
!-----------------------------------------------------------------------

      IF ( model_config_rec % mp_physics(1) .EQ. FAST_KHAIN_LYNN_SHPUND ) THEN 
         wrf_err_message = '--- ERROR: FAST SBM scheme must be built with a default compile-time flag'
         CALL wrf_message ( wrf_err_message )
         wrf_err_message = '--- ERROR: Run ./clean -a, ./configure, ./compile scripts again'
         CALL wrf_message ( wrf_err_message )
         count_fatal_error = count_fatal_error + 1
      END IF
# endif
```

Please remove those two lines in the `#else` branch in your PR.